### ISSUE TITLE
Basename prop doesn't work as expected

### ISSUE THREAD

**Issue by callumlocke** [supasate/connected-react-router#124]
I have followed the [FAQ to set a basename](https://github.com/supasate/connected-react-router/blob/master/FAQ.md#how-to-set-router-props-eg-basename-initialentries-etc) on my `history` object:

```jsx
import { createBrowserHistory } from 'history'

const history = createBrowserHistory({
  basename: '/prefix/',
})

// then...

<Provider store={store}>
  <ConnectedReactRouter history={history}>
	<App />
  </ConnectedReactRouter>
</Provider>
```

But when I connect `App` with react-redux, and do something like this:

```jsx
const mapStateToProps = state => ({
  pathname: state.router.location.pathname,
})
```

...the resulting `pathname` prop still includes the basename. I expected it to have no basename. Am I doing something wrong?

**Comment by callumlocke** [comment]
I can verify that dispatching a `push('/foo')` works correctly – it becomes `/prefix/foo` in the location bar.

But `state.router.location.pathname` still shows the prefix, which seems wrong.

When I use my `history` object directly (created with `createBrowserHistory`), if I do `history.location.pathname` it shows me whatever's in my browser location bar **minus the prefix**. This is correct behaviour. Why is connected-react-router not doing the same thing? Is it not actually using my passed history object? Is this a bug?

**Comment by callumlocke** [comment]
Upgrading fixed this. Seems to have been fixed somewhere between 4.3.0 to 4.4.1.
